### PR TITLE
keeping the names of the attributes same as platform for invoice view summary

### PR DESCRIPTION
**Pull request by sumitsourav**

### CODE REVIEW

File: app/platform_api/schema/invoice_summary.rb [hunk 1]
```diff
@@ -110,7 +110,7 @@ def self.id_ref
   # The outstanding balance amount in the group's base currency.
   # @type: number
   # @format: double
-  field :base_currency_outstanding_balance_amount
```
Comment (rishirajkumar97) [inline app/platform_api/schema/invoice_summary.rb:113]: should this be changed for invoice model as well?

Comment (sumitsourav) [reply app/platform_api/schema/invoice_summary.rb:113]: NO invoice model we are getting the response from v1/Invoices/query there we have the column name this only - "baseCurrencyOutstandingBalanceAmount" @rishirajkumar97

Comment (rishirajkumar97) [reply app/platform_api/schema/invoice_summary.rb:113]: hmm is platform fields are varying between Summary endpoints vs the actual invoice endpoint?

Comment (sumitsourav) [reply app/platform_api/schema/invoice_summary.rb:113]: Yes

**Pull request merged**